ETC.tsv

```diff
@@ -1462,17 +1462,17 @@ ETC_20150317_001461	Duration
 ETC_20150317_001462	User
 ETC_20150317_001463	Time Left
 ETC_20150317_001464	Name
-ETC_20150317_001465	{S18}{ol}{#ffffcc}Your name cannot exceed 16 characters {nl}including blank spaces.
+ETC_20150317_001465	{S18}{ol}{#ffffcc}Your name cannot exceed 16 characters {nl} including blank spaces.
```

Comment:
Why the space? A new line tag moves the text to the start of the line. Your edit would make it appear offset by one character
&nbsp;like this.